webrender/tests/angle_shader_validation.rs

```diff
@@ -16,8 +16,6 @@ struct Shader {
     features: &'static [&'static str],
 }
 
-const SHADER_PREFIX: &str = "#define WR_MAX_VERTEX_TEXTURE_WIDTH 1024\n";
```

Comment:
why does the shader validator need to have a separate list of shaders and features, given that we already have them described in RON?